deploy/Distech.CloudRelay.ARM/Gateway/Distech.CloudRelay.Gateway.json

```diff
@@ -28,13 +30,15 @@
       "minValue": 1,
       // maxValue not handled as it is a dynamic limit based on SKU
       "metadata": {
-        "description": "The number of of provisioned IoT Hub units"
+        "description": "The number of of provisioned IoT Hub units."
```

Comment:
Looks like we have a typo here too we could fix at the same time `number of of provisioned`